includes/rest-api/class-sensei-rest-api-lesson-quiz-controller.php

```diff
@@ -100,12 +100,15 @@ public function can_user_get_quiz( WP_REST_Request $request ) {
 	 * @return WP_REST_Response
 	 */
 	public function get_quiz( WP_REST_Request $request ) : WP_REST_Response {
-		$quiz = get_post( (int) $request->get_param( 'quiz_id' ) );
+		$lesson = get_post( (int) $request->get_param( 'lesson_id' ) );
+		$quiz   = Sensei()->lesson->lesson_quizzes( $lesson->ID );
 
-		$this->get_quiz_questions( $quiz );
+		if ( ! $quiz ) {
+			return new WP_REST_Response();
```

Comment:
Could we add a `204` (empty) status code here? Otherwise `apiFetch` would try to parse the empty response as JSON and fail.

```suggestion
			return new WP_REST_Response( null, 204 );
```